streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamTask.java

```diff
@@ -249,9 +249,11 @@ public ProcessorNode node() {
      * Commit the current task state
      */
     public void commit() {
+        log.trace("task [%s] flushing stateMgr", id());
```

Comment:
To be consistent with other logging entries (also `{}` is more efficient I think):

```
log.trace("task [{}] Start flushing all stores upon committing its state", id());
```

And ditto below:
```
log.trace("task [{}] Start flushing its producer's sent records upon committing its state", id());
```